### DISCUSSION TITLE
No response in backend

### DISCUSSION THREAD

**Discussion by notnvmm** [Atri-Labs/atrilabs-engine#652] [Help: Miscellaneous]
I'm facing the same issue. Updated the framework and started the local host again but no changes are being reflected on the output.

**Comment by cruxcode** [comment]
You will have to press Build & Run for changes to get reflected.

**Comment by notnvmm** [comment]
Sir
I did that. A couple of times. I even tried to change the values in the
code of the bar and made sure the name of bar chart is updated. Saved the
code and published the app again but no changes were reflected.

**Comment by darshitac11** [comment]
Please exit your current virtual environment, restart and then run `atri start --debug`. Please share a screen recording so that we may take a closer look.

**Comment by notnvmm** [comment]
I'm hereby attaching the respective screenshots if that's helpful. If it is
not clear, I can send the recording too.

On Tue, Feb 7, 2023 at 1:21 PM Darshita Chaturvedi ***@***.***>
wrote:

> Please exit your current virtual environment, restart and then run atri
> start --debug. Please share a screen recording so that we may take a
> closer look.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Atri-Labs/atrilabs-engine/discussions/652#discussioncomment-4890607>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AYMLQXIEWKKRZLNOMVHHOK3WWH5GHANCNFSM6AAAAAAUS55OC4>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>


-- 
Thanks
Shreya

**Reply by darshitac11** [reply]
I don't see a screenshot. Please attach it here.

**Reply by notnvmm** [reply]
Sorry for inconvenience. I'm attaching them here.

**Reply by notnvmm** [reply]
![Screenshot (31)](https://user-images.githubusercontent.com/102283357/217872251-168fe0d7-29f2-49d1-85fb-4c0d83133d5c.png)
![Screenshot (32)](https://user-images.githubusercontent.com/102283357/217872259-e937a2bb-5b4d-42f8-81ab-0da2dda5a62f.png)
![Screenshot (33)](https://user-images.githubusercontent.com/102283357/217872264-a5660c99-fa7e-4367-80b5-96233518fd2c.png)
![Screenshot (29)](https://user-images.githubusercontent.com/102283357/217872272-cb3e8135-25f1-4d56-9055-6353762aa9f9.png)

**Reply by cruxcode** [reply]
Don’t worry about Build and Run if it’s not working. We will judge based on how it looks in the editor.
We understand that the assignment is a bit demanding for students, hence, we would like to put off some burden.
Make sure you use correct CSS for positioning and aligning things.
Also make sure components are group together semantically.
These are the parameters on which we will judge.
